ExoCTK/contam_visibility/math_extensionsx.py

```diff
@@ -1,578 +1,611 @@
-#! /usr/bin/env python
-# math_extensions module
-
-"""This module provides simple extensions to the Python mathematical library."""
-
+"""This module provides simple extensions to the Python mathematical
+    library."""
+""" 2018/06/26 Made PEP8 compliant and added
+    sind() and cosd() - Joe Filippazzo"""
 """ Version 1 August 23, 2010 RLH - Added OBLIQUITY."""
-
 """ Version 0 August 6, 2010 RLH  - Created"""
 
-from math import *
-import copy
+from copy import deepcopy
+from math import radians, acos, asin, cos, sin, sqrt, pi, ceil, exp

```

Comment:
These imports should be one per line.

Reply:
Well, PEP8 says when calling explicit functions from a module, it's ook. Just don't call multiple modules in one line. https://www.python.org/dev/peps/pep-0008/?#imports

Reply:
WhoopsC3 -- I've been doing this wrong for a minute. Thanks!